### DISCUSSION TITLE
Mitigation steps for Talos published CVEs

### DISCUSSION THREAD

**Discussion by StayPirate** [CGAL/cgal#5947] [General]
Talos reported several security bugs in its last [vulnerability report](https://talosintelligence.com/vulnerability_reports/TALOS-2020-1225). They audited CGAL v5.1.1 and they have not mentioned anything about mitigation steps.
Can you please share the upstream point-of-view on these CVEs? Have they been addressed in any subsequent release? If yes, could you map fixing commit with the related CVE ID?

**Comment by lrineau** [comment]
Talos report was submitted in our issue tracker as https://github.com/CGAL/cgal/issues/5345, and it was fixed by the pull-request https://github.com/CGAL/cgal/pull/5371. That pull-request was in:
  - CGAL-5.1.3 and later releases,
  - CGAL-5.2.1, and later releases, and
  - CGAL-5.3.

Does that answer your question?

**Reply by StayPirate** [reply]
Yes, and thanks for the clarification.
